packages/x-components/static-docs/experience-search-and-discovery/empathize.md

```diff
@@ -36,10 +34,12 @@ search box.
 - [Recommendations](recommendations.md)
 - [History Queries](history-queries.md)
 - [Query Suggestions](query-suggestions.md)
+- [Next Queries](next-queries.md)
+- [ID Results](id-results.md)
 
```

Comment:
We should add TopClicked/Recommendations, as we have them in the Archetype:

![image](https://user-images.githubusercontent.com/4663897/186901496-f5a247e0-2537-4e31-bbff-84564d4d7000.png)

Reply:
We already have it! See the first item on the list! Or do you mean another one? @herrardo

Reply:
my bad!